README.md

```diff
@@ -5,7 +5,7 @@ Install into current virtualenv with
 
 ```
 $ pip install maturin
-$ maturin develop --release
+$ maturin develop --release --cargo-extra-args='--features extension-module,openssl'  # on Windows, drop `openssl`
```

Comment:
there is no way to make this the default? It seems like quite a long command line to enter

Reply:
I haven't been able to find one. I'm open to ideas.

Reply:
I don't see `openssl` declared as a "feature" in Cargo.toml. Are all dependencies implicitly features also, if you say "optional=true"?